src/main/g8/build.sbt

```diff
@@ -12,6 +12,16 @@ lazy val root = (project in file(".")).settings(
     // standard "effect" library (Queues, Console, Random etc.)
     "org.typelevel" %% "cats-effect-std" % "3.0.1"$if(scala3.truthy)$$else$,
     // better monadic for compiler plugin as suggested by documentation
-    compilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1")$endif$
-  )
+    compilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1")$endif$$if(testlib-use-cats-effect-testing-specs2.truthy)$,
+
+    "org.typelevel" %% "cats-effect-testing-specs2" % "1.0.0" % Test$else$
+
+    $endif$$if(testlib-use-munit-cats-effect-3.truthy)$,
+    "org.typelevel" %% "munit-cats-effect-3" % "1.0.0" % Test$else$$endif$
+
+  )$if(testlib-use-munit-cats-effect-3.truthy)$,
+
+  // this is needed to make munit suites collectable by sbt
+  testFrameworks += new TestFramework("munit.Framework")$else$$endif$
```

Comment:
Since [this PR](https://github.com/sbt/sbt/pull/6335) got merged, sbt 1.5.0 now supports munit out of the box and there's no need to specify it :)

Reply:
Thanks! I've updated the PR accordingly